csharp/.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+    "recommendations": [
+        "github.vscode-codeql",
+        "ms-dotnettools.csharp"
+        // "formulahendry.dotnet-test-explorer",
```

Comment:
ideally I would uncomment these, but I'm not sure of we want to recommend "random" extensions to contributors.

Reply:
@calumgrant Can I add these two recommendations without being commented out?

Reply:
I'm ok with these being commented out. Normally however we don't use code for TODOs, and use GitHub Issues instead.